druid/src/widget/button.rs

```diff
@@ -13,146 +13,98 @@
 // limitations under the License.
 
 //! A button widget.
-
 use crate::theme;
-use crate::widget::{Label, LabelText};
+use crate::widget::{Flex, Label, LabelText, MainAxisAlignment, Painter};
 use crate::{
-    Affine, BoxConstraints, Data, Env, Event, EventCtx, Insets, LayoutCtx, LifeCycle, LifeCycleCtx,
-    LinearGradient, PaintCtx, Point, Rect, RenderContext, Size, UnitPoint, UpdateCtx, Widget,
+    Data, Insets, LinearGradient, Point, Rect, RenderContext, UnitPoint, Widget, WidgetExt,
```

Comment:
I don't think that button should *require* the use of `WidgetExt`; really nothing should. What this means, I think, is that button should have its own `on_click` method; this can either make a clickable or we could still have our own logic for that?